api/src/main/resources/workbench.yaml

```diff
@@ -1938,6 +1951,24 @@ definitions:
         type: boolean
         description: Feature flag for enabling eRA commons
 
+  FeaturedWorkspacesConfigResponse:
+    type: object
+    properties:
+      featuredWorkspacesList:
+        type: "array"
+        items:
+          $ref: "#/definitions/FeaturedWorkspace"
+
+  FeaturedWorkspace:
+    type: object
+    properties:
+      name:
+        type: string
+        description: the name of the workspace
+      id:
```

Comment:
Is this sufficient? Shouldn't we need the namespace and the FC name? Maybe that's what is being referenced here, either way please name these more clearly

Reply:
"id" is the fc name; I don't think we need the namespace?

Reply:
i will add tho just in case